documentation/tutorials/AnalyzeOutput/modelVSdata.Rmd

```diff
@@ -31,62 +36,111 @@ Within both the run and out folders there will be one folder for each unique mod
 
   + Within this folder you will find, among other things, files of the format <year>.nc. Each of these files contains one year of model output in the standard PEcAn netCDF format.
 
-## Load and plot the model output
 
+## Load libraries and open connection to the database
 ```{r}
-library(PEcAn.utils)
-out <- read.output(99000000001, "output/PEcAn_99000000001/out/99000000001/" ,2004,2004,c("time","NEE"))
-plot(out$time,out$NEE,type='l',ylab="kg m-2 s-1",xlab="day of year")
+#Load every Library used
+library(PEcAn.all)
+library(PEcAn.DB)
+require(RPostgreSQL)
+library(PEcAn.assim.batch)
+library(PEcAn.benchmark)
 ```
 
-The arguments to read.output are the run ID, the folder where the run is located, the start year, the end year, and the variables being read.
+## Read in settings
 
-##	Compare model to flux observations
+There are two ways to access runs from pecan. One is from a settings object generated from an xml file, and the other is to query the database using the run ID, site ID, and output files manually. These ID's can all be found in the run README file. Both are shown below.  
 
-**First** _load up the observations_ and take a look at the contents of the file
+**From an XML file**
+```{r}
+## Read in the xml
+settings<-read.setting("filepath/custom.xml")
```

Comment:
probably will be easier for a new user if the file is pecan.xml, that way they only need to fill in the filepath